lib/shopify_app/controller_concerns/redirect_for_embedded.rb

```diff
@@ -16,7 +16,10 @@ def embedded_param?
 
     def redirect_for_embedded
       # Don't actually redirect if we're already in the redirect route - we want the request to reach the FE
-      redirect_to(redirect_uri_for_embedded) unless request.path == ShopifyApp.configuration.embedded_redirect_url
+      unless request.path == ShopifyApp.configuration.embedded_redirect_url
+        ShopifyApp::Utils::Logger.debug("Redirecting to #{redirect_uri_for_embedded}")
+        redirect_to(redirect_uri_for_embedded)
+      end
```

Comment:
I kind of started going a little wild and logging every redirect, then stopped and figured I would see what other people think. Should I just log every redirect? Does it matter if the redirect is in a private method?